recipes/asap/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "asap" %}
+{% set version = "3.10.0" %}
+{% set sha256 = "d58105290029385835569e1e7057d7b03bad1dd0c5efc2151370e77f7721f478" %}
+{% set asap_version = "3_10_0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-ASAP_VERSION_{{ asap_version}}-a78278070c997b302c8117d8174c8914d5f3d273.tar.gz
+  url: https://gitlab.com/{{ name }}/{{ name }}/repository/archive.tar.gz?ref=ASAP_VERSION_{{ asap_version }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --record record.txt
```

Comment:
That seems to cause issues with the build. `error: option --single-version-externally-managed not recognized`